packages/bits/src/lib/button/button.component.html

```diff
@@ -2,6 +2,9 @@
     *ngIf="isBusy"
     [ngStyle]="getRippleContainerStyle()"
     class="nui-button-ripple-container"
+    aria-live="polite"
+    aria-atomic="true"
```

Comment:
aria-atomic true I wouldn't use that because it force screen readers to read it contents